app/views/taxons/show.html.erb

```diff
@@ -38,17 +38,17 @@
   <nav role="navigation" class="taxon-page__grid">
     <div class="full-page-width-wrapper">
       <h2 class="taxon-page__section-heading taxon-page__grid-heading">Explore these sub-topics</h2>
-      <ol>
+      <ol class="taxon-page__grid-wrapper">
         <% presented_taxon.child_taxons.each_with_index do |child_taxon, index| %>
-          <li class="<%= 'leftmost-row-cell-clear-float' if index % 3 == 0 %>">
+          <li class="taxon-page__grid-item">
             <h2>
```

Comment:
Previously mentioned but would be good to have an explicit class on this heading for styling purposes.

Reply:
Good point, I think this is a bit confusing as the same styles are being applied to worldwide pages too, but this isn't immediately obvious from the code. I'll try and tidy it up 👍

Reply:
Done here: https://github.com/alphagov/collections/pull/624/commits/6a0c3a25f6c08cf8b2bdec62e59c089af4858545